code/game/objects/effects/weeds.dm

```diff
@@ -1,5 +1,5 @@
 //Color variant defines
-#define SPEED_COLOR ""
+#define NORMAL_COLOR ""
```

Comment:
this should really be like, an actual define, not just a blank. Not sure if you'll need to update anything else to take this into account though.